.rubocop.yml

```diff
@@ -49,3 +54,7 @@ Style/StringLiterals:
 # Configuration parameters: WordRegex.
 Style/WordArray:
   Enabled: false
+
+# we intentionally ignore some exceptions in fork as we have no way to report it back
```

Comment:
NP: This applies to all exceptions. Better disable the cop in the specific place where we've checked it is safe.